src/Stratis.Bitcoin.IntegrationTests/ConsensusManagerTests.cs

```diff
@@ -0,0 +1,93 @@
+﻿using System;
+using NBitcoin;
+using Stratis.Bitcoin.IntegrationTests.Common;
+using Stratis.Bitcoin.IntegrationTests.Common.EnvironmentMockUpHelpers;
+using Stratis.Bitcoin.Networks;
+using Xunit;
+
+namespace Stratis.Bitcoin.IntegrationTests
+{
+    public class ConsensusManagerTests
+    {
+        private readonly Network network;
+
+        private const string walletAccount = "account 0";
+        private const string walletName = "mywallet";
+        private const string walletPassword = "123456";
+
+        public ConsensusManagerTests()
+        {
+            this.network = new StratisRegTest();
+            this.network.Consensus.Options = new ConsensusOptionsTest();
+
+            Type consensusType = typeof(NBitcoin.Consensus);
+            consensusType.GetProperty("MaxReorgLength").SetValue(this.network.Consensus, (uint)20);
+        }
+
+        private class ConsensusOptionsTest : PosConsensusOptions
+        {
+            public override int GetStakeMinConfirmations(int height, Network network)
+            {
+                return height < 10 ? 5 : 10;
+            }
+        }
+
+        [Fact]
+        public void ConsensusManager_Fork_Scenario_1()
```

Comment:
ConsensusManager_Fork_Scenario_1 is referring to the test scenario?
Better is to describe the test in the method name and in the comment write the scenario number (as we did with CHT)

Reply:
Will update :)